### DISCUSSION TITLE
Recommended board

### DISCUSSION THREAD

**Discussion by glenosAU** [vitotai/BrewPiLess#413] [Q&A] [answered]
My D1 mini pro board has failed, seems to be a hardware problem I cannot connect to it using esptool.py. So I am looking for a replacement board. Are there any recommended boards or should I just get a D1 mini pro again?

Thanks
Glen

**Comment by vitotai** [comment, marked as answer]
Get an Esp32. I am turning to ESP32 and will stop supporting ESP8266 some time later. 
Supporting devices that I don't use makes no sense for me and makes little for others. It's difficult to maintain since I would not be able to know issues on the devices.

**Reply by glenosAU** [reply]
Any particular ESP32?
ESP32-C3, ESP32-S3 (mini/pro)?
https://www.wemos.cc/en/latest/index.html

**Reply by vitotai** [reply]
Regular ESP32? Not C3, S3 or other model. Just plain ESP32.

**Reply by glenosAU** [reply]
I got a ESP32-WROOM it is working fine.

**Comment by ufacek** [comment]
last recommended board is still ESP32 or is it different now?
like this one https://www.hadex.cz/m432f-esp-wroom-32-vyvojova-deska-24ghz-wifibluetooth/
Thanks
